docs/guides/schemas.md

```diff
@@ -112,3 +112,67 @@ This validation defines a very specific (honestly, useless) behavior, where if a
 When you need this level of specificity, using the `validateNode` property of the editor or plugins is handy.
 
 However, only use it when you absolutely have to. And when you do, you need to be aware of its performance. `validateNode` will be called **every time the node changes**, so it should be as performant as possible. That's why the example above returns early, so that the smallest amount of work is done as possible each time it is called.
+
+## Multi-step Normalizations
+
+Some normalizations will require multiple `change` function calls in order to complete. 
```

Comment:
Add after this sentence:

> But after calling the first change function, the resulting document will be normalized, changing it out from under you. This can cause intended behaviors.